hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AEncryptionDSSEKMSUserDefinedKey.java

```diff
@@ -39,10 +42,12 @@ protected Configuration createConfiguration() {
     Configuration c = new Configuration();
     String kmsKey = S3AUtils.getS3EncryptionKey(getTestBucketName(c), c);
     // skip the test if DSSE-KMS or KMS key not set.
-    if (StringUtils.isBlank(kmsKey)) {
-      skip(S3_ENCRYPTION_KEY + " is not set for " +
-          DSSE_KMS.getMethod());
+    try {
+      skipIfEncryptionNotSet(c, DSSE_KMS);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
```

Comment:
if this was production i'd ask for an UncheckedIOException(); not so worried here